MiniDumper/CommandLineOptions.cs

```diff
@@ -28,6 +28,9 @@ class CommandLineOptions
         [Option('l', HelpText = "Display the debug logging of the process + diagnostics info from the minidumper.")]
         public bool Verbose { get; set; }
 
+        [Option('p', HelpText = "Memory commit threshold in MB at which to create a dump.")]
```

Comment:
I would prefer **-m** option (or **-mx** as we might have problems with a single-letter option in the CommandLineParser). If possible, please also add the **-ml** option to make dump when the memory usage gets below the specified threshold.